tests/data/EraseOpenCornersIncompatibilityTest-Two.ufo/glyphs/incompatible.glif

```diff
@@ -1,20 +1,20 @@
-<?xml version='1.0' encoding='UTF-8'?>
+<?xml version="1.0" encoding="UTF-8"?>
 <glyph name="incompatible" format="2">
-  <advance width="652"/>
-  <outline>
-    <contour>
-      <point x="464" y="189" type="curve"/>
-      <point x="494" y="359" type="line"/>
-      <point x="348" y="256" type="line"/>
-      <point x="340" y="279" type="line"/>
-      <point x="399" y="268"/>
-      <point x="428" y="235"/>
-    </contour>
-  </outline>
-  <lib>
-    <dict>
-      <key>com.schriftgestaltung.Glyphs.lastChange</key>
-      <string>2021/09/22 09:53:10</string>
-    </dict>
-  </lib>
+	<advance width="652"/>
+	<outline>
+		<contour>
+			<point x="464" y="189" type="curve"/>
```

Comment:
why the inconsistent tap/space indentation here? I suggest you rewrite the test ufo so they match the original 2-space indent (open and save back using ufoLib2)

Reply:
nevermind sorry, I was misled by Github's UI that showed the first indent level narrower than the second, making me believe you were mixing tab and spaces. I don't actually care whether you use one or the other, just not both. Ignore my comment